src/NServiceBus.Msmq.AcceptanceTests/When_distributing_an_event.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AcceptanceTesting.Customization;
+    using AcceptanceTesting;
+    using Features;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+    using Support;
+
+    public class When_distributing_an_event : NServiceBusAcceptanceTest
+    {
+        static string PublisherEndpoint => Conventions.EndpointNamingConvention(typeof(Publisher));
+        static string SubscriberEndpoint => Conventions.EndpointNamingConvention(typeof(Subscriber));
+
+        [Test]
+        public async Task Should_round_robin()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Publisher>(b => b.When(c => c.Subscriptions == 2, async (session, c) =>
+                {
+                    for (var i = 0; i < 10; i++)
+                    {
+                        await session.Publish(new MyEvent()).ConfigureAwait(false);
+                    }
+                }))
+                .WithEndpoint<Subscriber1>()
+                .WithEndpoint<Subscriber2>()
+                .Done(c => c.Receiver1TimesCalled + c.Receiver2TimesCalled>= 10)
+                .Run(TimeSpan.FromSeconds(20));
+
+            Assert.That(context.Receiver1TimesCalled + context.Receiver2TimesCalled, Is.EqualTo(10));
+        }
+
+        public class Context : ScenarioContext
+        {
+            public int Subscriptions { get; set; }
+            public int Receiver1TimesCalled => receiver1TimesCalled;
+            public int Receiver2TimesCalled => receiver2TimesCalled;
```

Comment:
If the fields are already public, is there much point in having these properties?